Launcher/Classes/ViewController.swift

```diff
@@ -684,5 +696,45 @@ class TasksViewController: NSViewController {
             strongSelf.createIRBSessionTask.waitUntilExit()
         }
     }
+    
+    func getCurrentDeviceUDID() -> String {
+        let simulatorUDIDs = "\(phoneComboBox.itemTitle(at: phoneComboBox.indexOfSelectedItem))"
```

Comment:
Why do you wrap this in a String literal yourself? Doesn't `itemTitle(at:_)` already return a `String`?